ansible_collections/arista/avd/plugins/plugin_utils/schema/avdschematools.py

```diff
@@ -177,16 +177,6 @@ def handle_validation_exceptions(self, exceptions: Generator, mode: str | None)
                 self.ansible_display.error(message, wrap_text=False)
         return counter
 
-    def validate_schema(self) -> int:
```

Comment:
TODO - check why this is removed in this PR

Reply:
It was just removed because it is unused and showed up in my editor.